examples/acceptFromIDs.js

```diff
@@ -1,6 +1,16 @@
-var SteamUser = require('steam-user');
-var client = new SteamUser();
-var TradeOfferManager = require('steam-tradeoffer-manager');
+const SteamUser = require('steam-user'),
```

Comment:
The problem is the other examples (and the most of the code) use `var` declarations.